stubs/python-slugify/slugify/__init__.pyi

```diff
@@ -1,2 +1,12 @@
+from .__version__ import (
```

Comment:
Shouldn't these all be explicit re-exports (`__author__ as __author__`, etc)?

Reply:
Thanks, they should. I started off with `from .__version__ import *`, but that didn't work and I lost the re-export when fixing that